### PR TITLE
Feature/onboarding/claiming nodes

### PR DESCRIPTION
**Pull request by jjtsou**

### CODE REVIEW

File: src/components/tabs/tab.js [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 import { StyledTab } from "./styled"
 
-export const Tab = ({ label, index, onChange, ...rest }) => {
+export const Tab = ({ index, onChange, ...rest }) => {
   const onClick = useCallback(() => onChange && onChange(index || 0), [index, onChange])
 
   return (
     <StyledTab onClick={rest.disabled ? undefined : onClick} {...rest}>
-      {label}
+      {rest.label}
```
Comment (jacekkolasa) [inline src/components/tabs/tab.js:9]: out of curiosity, this change will pass `label` prop to StyledTab, is this intentional?

**Review by jacekkolasa — APPROVED** [review]

**Comment by jacekkolasa** [comment]
@jjtsou i published it so it can be used in https://github.com/netdata/cloud-frontend/pull/2682

File: src/components/tabs/tab.js (hunk 1 above)
Comment (jjtsou) [reply src/components/tabs/tab.js:9]: Yes, this change was intentional. [For reference](https://github.com/netdata/cloud-frontend/pull/2681/files#diff-052cc627f52854365d9eba7cdbb3ab0fd1a9bb845e3d1b8c328c37467c0cfac9)

Comment (jjtsou) [reply src/components/tabs/tab.js:9]: ` <Tab label={<TabLabel label="linux" />} as={StyledButton}>`
Button takes it's label as a prop.

Comment (jacekkolasa) [reply src/components/tabs/tab.js:9]: @jjtsou thanks!

**Pull request merged**